lib/kubeclient/common.rb

```diff
@@ -223,13 +219,10 @@ def create_entity(entity_type, entity_config, klass)
 
     def update_entity(entity_type, entity_config)
       name = entity_config.metadata.name
-      # to_hash should be called because of issue #9 in recursive open
-      # struct
-      hash = entity_config.to_hash
-      ns_prefix = build_namespace_prefix(entity_config.metadata['table'][:namespace])
+      ns_prefix = build_namespace_prefix(entity_config.metadata.namespace)
```

Comment:
@bkoski please don't remove the access to `namespace` using `['table'][:namespace]`, it's going to create a regression in applications where `.namespace` is already overridden.

Was that required for 2.3.0?